project-fob/Controllers/Classes/MeetingWrapper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using project_fob.Data;
+using project_fob.Models;
+
+namespace project_fob.Controllers
+{
+    public static class MeetingWrapper
+    {
+        public static Meeting CreateMeeting(string meetingId, string attendeePassword, string hostPassword, ApplicationDbContext db)
+        {
+            Meeting meeting = new Meeting { MeetingId = meetingId, RoomPassword = attendeePassword, HostPassword = hostPassword, Active = true };
+            db.Meeting.Add(meeting);
+            return meeting;
+        }
+
+        public static void UserPressedFob(Meeting meeting, Guid id, ApplicationDbContext db)
+        {
+            meeting.Fobbed.Add(db.User.Find(id) ?? new User { Id = id });
```

Comment:
yes... but oddly enough it works fine ...

Reply:
I wonder if it looks OK because somewhere it's not leading the Fobbed list, so it's starting with an empty collection. If that is the case, then it'll probably lead to bugs if you have multiple people in the same meeting trying to hit Fob at the same time.

One way to check this would be to change the Meeting model so that Fobbed doesn't start off as empty list - leave it as `null` if it's not loaded from the database (which is probably a good idea anyway)

Reply:
Removing the Fobbed list initializing (new list...) doesn't affect the app with a new db.
setting the Fobbed list to Null (= NULL / not putting anything) doesn't crash the app directly. It crashes the app only when the host is clicking on the finish meeting

The funny part is that the counter seems to work fine (even with multiple users) but when we want to pass the fobbed count to stats it crashing.